microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation.md

```diff
@@ -30,7 +30,7 @@ For a webpage, you can use the [prefers-color-scheme][MDNPrefersColorScheme] CSS
     :::image-end:::  
```

Comment:
At top of article, change:
Having a web product that has a light theme in a dark-mode operating system is grating
to:
Having a web product that has a light theme in a dark-mode operating system can be hard to read